src/commands/basic/help.js

```diff
@@ -1,84 +1,94 @@
       const categories = readdirSync("./commands/");
 
       embed.setDescription(
-        `These are the avaliable commands for ${message.guild.me.displayName}\nThe bot prefix is: **${client.config.prefix}**`
+        `These are the avaliable commands for ${message.guild.me.displayName}\nThe bot prefix is: **${this.client.config.prefix}**`
       );
       embed.setFooter(
-        `© ${message.guild.me.displayName} | Developed by DistroByte | Total Commands: ${client.commands.size}`,
-        client.user.displayAvatarURL()
+        `© ${message.guild.me.displayName} | Developed by DistroByte | Total Commands: ${this.client.commands.size}`,
+        this.client.user.displayAvatarURL()
       );
 
       categories.forEach((category) => {
-        const dir = client.commands.filter(
-          (c) => c.config.category === category
+        const dir = this.client.commands.filter(
+          (c) => c.help.category === category
         );
         const capitalise =
           category.slice(0, 1).toUpperCase() + category.slice(1);
         try {
           embed.addField(
             `${capitalise} [${dir.size}]:`,
-            dir.map((c) => `\`${c.config.name}\``).join(" ")
+            dir.map((c) => `\`${c.help.name}\``).join(" ")
           );
-        } catch (e) {}
+        } catch (e) {  }
       });
 
       return message.channel.send(embed);
     } else {
-      let command = client.commands.get(
-        client.aliases.get(args[0].toLowerCase()) || args[0].toLowerCase()
+      let command = this.client.commands.get(
+        this.client.aliases.get(args[0].toLowerCase()) || args[0].toLowerCase()
       );
       if (!command)
         return message.channel.send(
           embed
             .setTitle("Invalid Command.")
             .setDescription(
-              `Do \`${client.config.prefix}help\` for the list of the commands.`
+              `Do \`${this.client.config.prefix}help\` for the list of the commands.`
             )
         );
       command = command.config;
 
-      embed.setDescription(stripIndents`The bot's prefix is: \`${client.config.prefix}\`\n
-            **Command:** ${
-              command.name.slice(0, 1).toUpperCase() + command.name.slice(1)
-            }
-            **Description:** ${
-              command.description || "No Description provided."
-            }
-            **Usage:** ${
-              command.usage
-                ? `\`${client.config.prefix}${command.name} ${command.usage}\``
-                : `\`${client.config.prefix}${command.name}\``
-            }
+      embed.setDescription(stripIndents`The bot's prefix is: \`${this.client.config.prefix}\`\n
+            **Command:** ${command.name.slice(0, 1).toUpperCase() + command.name.slice(1)
+        }
+            **Description:** ${command.description || "No Description provided."
+        }
+            **Usage:** ${command.usage
+          ? `\`${this.client.config.prefix}${command.name} ${command.usage}\``
```

Comment:
Almost all useage info are arrays of strings which start with `{{p}}` is this part ment to be stripted, and will this work if its an array of strings?

Reply:
This is a screwup on my end. It's supposed to be different with the new commands. Fixed.